moto/wafv2/responses.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import unicode_literals
+
+import json
+
+from moto.core.responses import BaseResponse
+from .models import wafv2_backends, GLOBAL_REGION
+
+
+class WAFV2Response(BaseResponse):
+    SERVICE_NAME = "wafv2"
+
+    @property
+    def wafv2_backend(self):
+        return wafv2_backends[self.region]
+
+
+class WafV2Handler(BaseResponse):
```

Comment:
These two classes can be combined (`WAFV2Response`/`WavV2Handler`)

Reply:
Combined into `WAFV2Response`